llava/eval/model_vqa_science.py

```diff
@@ -97,6 +100,7 @@ def eval_model(args):
     parser = argparse.ArgumentParser()
     parser.add_argument("--model-path", type=str, default="facebook/opt-350m")
```

Comment:
Can you change line 101 and 102 based on https://github.com/nahidalam/LLaVA/blob/maya_eval/llava/eval/model_vqa_mmbench.py#L146-L147